### PR TITLE
Fix: Remove unused parameters

### PR DESCRIPTION
**Pull request by localheinz**
This PR

* [x] removes unused parameters

### CODE REVIEW

File: tests/Solarium/Tests/Core/Client/ClientTest.php [hunk 1]
```diff
@@ -551,7 +551,7 @@ public function testCreateRequest()
              ->method('getRequestBuilder')
              ->will($this->returnValue($observer));
 
-        $this->client->registerQueryType('testquerytype', 'Solarium\QueryType\Select\Query\Query', $observer, '');
+        $this->client->registerQueryType('testquerytype', 'Solarium\QueryType\Select\Query\Query');
```
Comment (localheinz) [inline tests/Solarium/Tests/Core/Client/ClientTest.php]: See [`Solarium\Core\Client\ClientInterface::registerQueryType()`](https://github.com/solariumphp/solarium/blob/91fe40852783f16a5909064a400add33ffef1c52/library/Solarium/Core/Client/ClientInterface.php#L193).

File: tests/Solarium/Tests/QueryType/Select/Query/AbstractQueryTest.php [hunk 2]
```diff
@@ -262,7 +262,7 @@ public function testAddAndGetFilterQueryWithKey()
     {
         $key = 'fq1';
 
-        $fq = $this->query->createFilterQuery($key, true);
+        $fq = $this->query->createFilterQuery($key);
```
Comment (localheinz) [inline tests/Solarium/Tests/QueryType/Select/Query/AbstractQueryTest.php]: See [`Solarium\QueryType\Select\Query\Query::createFilterQuery()`](https://github.com/solariumphp/solarium/blob/0acdb2838ae551fe87e4e334e449a468973839f2/library/Solarium/QueryType/Select/Query/Query.php#L602).

File: tests/Solarium/Tests/QueryType/Select/Query/AbstractQueryTest.php [hunk 3]
```diff
@@ -612,7 +612,7 @@ public function testRegisterComponentType()
         $components = $this->query->getComponentTypes();
         $components['mykey'] = 'mycomponent';
 
-        $this->query->registerComponentType('mykey', 'mycomponent', 'mybuilder', 'myparser');
+        $this->query->registerComponentType('mykey', 'mycomponent');
```
Comment (localheinz) [inline tests/Solarium/Tests/QueryType/Select/Query/AbstractQueryTest.php]: See [`Solarium\QueryType\Select\Query\Query::registerComponentType()`](https://github.com/solariumphp/solarium/blob/0acdb2838ae551fe87e4e334e449a468973839f2/library/Solarium/QueryType/Select/Query/Query.php#L764)

File: tests/Solarium/Tests/QueryType/Select/Query/Component/Stats/StatsTest.php [hunk 4]
```diff
@@ -139,7 +139,7 @@ public function testAddAndGetFieldWithKey()
     {
         $key = 'f1';
 
-        $fld = $this->stats->createField($key, true);
+        $fld = $this->stats->createField($key);
```
Comment (localheinz) [inline tests/Solarium/Tests/QueryType/Select/Query/Component/Stats/StatsTest.php]: See [` Solarium\QueryType\Select\Query\Component\Stats\Stats::createField()`](https://github.com/solariumphp/solarium/blob/91fe40852783f16a5909064a400add33ffef1c52/library/Solarium/QueryType/Select/Query/Component/Stats/Stats.php#L114).

File: tests/Solarium/Tests/QueryType/Update/RequestBuilderTest.php [hunk 5]
```diff
@@ -437,11 +437,9 @@ public function testBuildCommitXmlWithParams()
 
     public function testBuildRollbackXml()
     {
-        $command = new RollbackCommand;
-
         $this->assertEquals(
             '<rollback/>',
-            $this->builder->buildRollbackXml($command)
+            $this->builder->buildRollbackXml()
```
Comment (localheinz) [inline tests/Solarium/Tests/QueryType/Update/RequestBuilderTest.php]: See [`Solarium\QueryType\Update\RequestBuilder::buildRollbackXml()`](https://github.com/solariumphp/solarium/blob/0acdb2838ae551fe87e4e334e449a468973839f2/library/Solarium/QueryType/Update/RequestBuilder.php#L212).

**Comment by basdenooijer** [comment]
Thanks a lot for all your PRs, I'll try to work through them in the next few days

**Comment by localheinz** [comment]
Thank you, @basdenooijer!

**Pull request merged**